cfpb/design-system
# Add Card layout CSS and docs

## Additions

- Adds `o-card-group` and `m-card` components to `cfpb-layout` package.

## Testing

1. Pull branch, run `cd docs && yarn && yarn start`
2. Check http://localhost:4000/design-system/components/cards

## Screenshots

![cards-ds](https://user-images.githubusercontent.com/704760/75469516-26712000-595d-11ea-8a56-73e01a79db44.png)

Hover:

![ask-card-hover](https://user-images.githubusercontent.com/704760/75469866-b8792880-595d-11ea-92c0-0409851153ff.gif)

![card-hover](https://user-images.githubusercontent.com/704760/75469868-b911bf00-595d-11ea-9500-2938b7d980ab.gif)

Focus:

![ask-card-focus](https://user-images.githubusercontent.com/704760/75469882-c038cd00-595d-11ea-8dfb-563ffab3690e.gif)

![card-focus](https://user-images.githubusercontent.com/704760/75469883-c038cd00-595d-11ea-8e29-d97b83118853.gif)
